eq-author/src/middleware/headers/versionHeader.js

```diff
@@ -0,0 +1,4 @@
+export default headers => ({
+  ...headers,
+  clientVersion: process.env.REACT_APP_EQ_AUTHOR_VERSION,
```

Comment:
Probably best to only set the request header if `REACT_APP_EQ_AUTHOR_VERSION` has a value.

Reply:
![image](https://user-images.githubusercontent.com/4097796/53101505-cf529700-3521-11e9-811a-357a11300bb1.png)